### ISSUE TITLE
Core Updater: not showing cores

### ISSUE THREAD

**Issue by bitshiftnetau** [libretro/RetroArch#11288]
## Description

Core Updater shows no cores. Setting core_updater_show_experimental_cores = "true" shows experimental cores with random names e.g. "3dengine_libretro.so.zip" instead of something like "NES / Famicom (Nestopia EU)" 

Tried changing DNS and nothing. No firewall or VPN set on this network or machine. Had to change libretro_directory to home config coz Arch defaults to a system folder for pacman, but that only fixed downloading the experimental cores that I can see.

### Expected behavior

I expected to see the cores with typical names like in most of the tutorials. I'm not seeing that.

### Actual behavior

Cores aren't displaying. Experimental ones are displaying.

### Steps to reproduce the bug

1. Install on Arch Linux
2. Enable Core Updater in config file
3. Open Core Updater

### Bisect Results

I'm new to this program, unfortunately I have no idea if this has started happening recently or when that might have been.

### Version/Commit
-RetroArch: 1.9.0/5e551dd92b

### Environment information

- OS: ArchLinux kernel 5.4.62-1-lts
- Compiler: n/a

**Comment by hizzlekizzle** [comment]
Are you missing the core info files? You should still be able to see them using the lib names even without them, though.

Can you try this AppImage build and see if it treats you any differently? https://github.com/hizzlekizzle/RetroArch-AppImage/releases/tag/Nightlies

**Comment by bitshiftnetau** [comment]
Sure, I've tried to launch that but there are simply too many missing libraries. Is there a way to launch this without having to symlink hundreds of packages to older versions? Seems like it defeats the whole point of .AppImage files in the first place.

**Comment by hizzlekizzle** [comment]
Yes, apparently AppImages don't bundle a number of important libs because they expect some of them to already exist on the host machine. You can try this one and see if it treats you any better: https://github.com/hizzlekizzle/RetroArch-AppImage/releases/download/Linux_LTS_Nightlies/RetroArch-Linux-x86_64-Nightly.AppImage

I only suggested the AppImage because it's the least intrusive "universal" package, but we also have a snap and flatpak if you'd care to try either of those.

**Comment by bitshiftnetau** [comment]
Hey sorry been flat out, I'm still on this issue and will try the AppImage when I get a chance.

**Comment by gouchi** [comment]
@bitshiftnetau Please close this issue, if you don't reproduce the issue.

If you installed RetroArch from Arch, you will have to install first [libretro-core-info](https://archlinux.org/packages/extra/any/libretro-core-info/) then install [libretro cores](https://archlinux.org/packages/?sort=&q=libretro-&maintainer=&flagged=).

If you did not install from the package manager, you will have to update core info first to use the online updater.

Thank you.
